src/app/views/infoblock/infoblock.js

```diff
@@ -0,0 +1,48 @@
+class InfoblockController {
+
+  /** @ngInject */
+  constructor($log) {
+    this.$log = $log;
+  }
+
+  $onChanges() {
+    if (!this.variable) {
+      return;
+    }
+
+    this.value = this.variable.value;
+    this.title = this.variable.title;
+
+    this.colorClass = '';
+    if (this.colorOverride) {
+      this.colorClass = `color-${this.colorOverride}`;
+    } else if (this.isChange && this.value !== 0) {
+      let ispositive = (this.value > 0);
+      if (this.reverseColor) {
+        ispositive = !ispositive;
+      }
+      this.colorClass = ispositive ? 'color-positive' : 'color-negative';
+    }
+
+    this.value = Math.round(this.value * 100) / 100;
```

Comment:
Number won't touch strings (It'll just give a blank result), and for any numbers it'll tack on as many zeros as it needs to get exactly that many fractional digits, so like 12.3 will become 12.30.

Looking into localizing percents it seems like some locales put it on the left side (Can't find about Mongolian specifically), not sure about the '+' character. There is [Number.toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) but that has poor IE support and no Safari support :(